region/brinstar/pink.json

```diff
@@ -1385,6 +1385,76 @@
+                  "requires": [
+                    "Morph",
+                    "canWallIceClip",
+                    "Wave"
+                  ],
+                  "clearsObstacles": ["C"],
+                  "note": [
+                    "Freeze a Zeb inside of the crumble blocks below the entrance to the Power Bomb room and use it to get stuck inside the crumble blocks, breaking them.",
+                    "Repeatedly freeze a left facing Zeb from the enemy spawner lower in the room.",
+                    "Freeze it close to the right side of the crumble block so that Samus will attach to the enemy when rolling into it.",
+                    "Once the Zeb unfreezes, or dies, the crumble block will break."
+                  ]
+                }
+              ]
+            },
+            {
+              "id": 5,
+              "strats": [
+                {
+                  "name": "Big Pink Reo Ice Clip",
+                  "notable": true,
+                  "requires": [
+                    "canTrickyUseFrozenEnemies",
+                    "canStaggeredWalljump",
+                    "canXRayCeilingClip",
+                    {"or": [
+                      {"and": [
+                        "Morph",
+                        "canXRayStandUp"
+                      ]},
+                      "canPreciseCeilingClip"
+                    ]}
+                  ],
+                  "note": [
+                    "Lure the Reo (Bee) to the left to use it as a platform for ice clipping up through the crumbles to the Power Bomb room exit door.",
+                    "Watch the Reo bounce against the left wall until it does a slow hover towards Samus.",
+                    "There is a very small timing of when its right claw is above the small pale spore next to the big red spore, when Samus should jump to be above the Reo.",
+                    "Positioning Samus' front toe beneath this spore (facing left) will cause the bee to fly with the correct timing after Samus takes knockback damage, with a 3 pixel window.",
+                    "This will cause it to fly left and into the two tile gap below the crumble blocks.",
+                    "Freeze it once to get above it, and then again for the ice clip.",
+                    "Once it is up there, the bee will become stuck and the room will need to be reset.",
+                    "But if the Reo does not fit in the hole, it is possible to just wait near the ledge for another slow hover to try again."
+                  ]
+                },
+                {
+                  "name": "Big Pink Zeb Ice Clip",
+                  "notable": true,
+                  "requires": [
+                    "canTrickyUseFrozenEnemies",
+                    "canStaggeredWalljump",
+                    "canCameraManip",
+                    {"or": [
+                      "Wave",
+                      "Spazer"
+                    ]},
+                    "canPreciseCeilingClip"
```

Comment:
I would assume x-ray and morph can help here too?
```suggestion
                    "h_canNonTrivialCeilingClip"
```

Reply:
You would have to crouch->shoot->jump to get the right height, and idk if its possible with that, but maybe.  I haven't seen a setup for that anyway, and you need a setup for this trick.